src/advert/adservers.txt

```diff
@@ -154,6 +154,7 @@
 ||jillsclickcorner.com^$third-party
 ||jnwsrqxrdqct.top^$popup
 ||juruiklan.com^$third-party
+||kambingjantan.cc^$third-party
```

Comment:
`kambingjantan.cc` is image host provider like imgur

Reply:
Hey @gmarcher i didn't know that, thank you for the advise. 
What do you think about : `||kambingjantan.cc/storage/drive$domain=manhwalands.xyz` Moving to specific block filter ?